EFAuditer/AuditDatabaseContext.cs

```diff
@@ -8,42 +8,28 @@ namespace EFAuditer
 {
     public class AuditDatabaseContext : DbContext
     {
+        public AuditDatabaseContext() {}
         public AuditDatabaseContext(DbContextOptions<AuditDatabaseContext> options) : base(options) {}
 
         public virtual DbSet<AuditLog> AuditLogs { get; set; }
 
-        public async Task AuditReadOperationAsync(string primaryKeyName, int primaryKeyId, object model)
+        public async Task AuditOperationAsync(int primaryKeyId, string entity, string details, string eventType)

```

Comment:
Decided it made more sense to keep this general rather than tied to a "Read" event